### PR TITLE
Behaviour changes for Non admin role

### PR DESCRIPTION
**Pull request by SadafAttar**
- Add UI for behaviours in non-admin mode
- Refactor Behaviour component

In an earlier [PR](https://github.com/iTwin/iot-demo/pull/163), we added a feature to define behavior (simulated data pattern) for IoT devices for admin users. Because only admin users can add/edit behavior for a device. Non-admin users can only view the already-defined device behavior. So these changes are required for non-admin users to appropriately see the device behavior in the device list.

### CODE REVIEW

File: IoTDeviceSimulator/web-client/src/BehaviourComponent.js [hunk 1]
```diff
@@ -43,14 +43,32 @@ export function SineComponent(props) {
     }
     return (
         <>
-            <div className="behaviour-value">
-                <LabeledInput className="labels" type='number' label='Mean' name='mean' value={deviceMean} onChange={changeMean} disabled={props.signalArray !== "" ? true : false} />
-                <LabeledInput className="labels" type='number' label='Amplitude' name='amplitude' value={deviceAmplitude} onChange={changeAmplitude} disabled={props.signalArray !== "" ? true : false} />
-                <LabeledInput className="labels" type='number' label='Period(ms)' name='wave_period' value={deviceWavePeriod} onChange={changeWave_period} disabled={props.signalArray !== "" ? true : false} />
-            </div >
-            <ChartComponent labelsArray={arr} dataArray={ar} chartName="Component Signal" />
+            {props.isAdmin === true ?
```
Comment (SheetalNair) [inline IoTDeviceSimulator/web-client/src/BehaviourComponent.js:46]: There are conditional renders in each component for non admin user, can we avoid that and use signalArray to create a table

File: IoTDeviceSimulator/web-client/src/BehaviourComponent.js [hunk 2]
```diff
@@ -43,14 +43,32 @@ export function SineComponent(props) {
...
+            <>
+                <div className="behaviour-value">
+                    <LabeledInput className="labels" type='number' label='Mean' name='mean' value={deviceMean} onChange={changeMean} disabled={props.signalArray !== "" ? true : false} />
+                    <LabeledInput className="labels" type='number' label='Amplitude' name='amplitude' value={deviceAmplitude} onChange={changeAmplitude} disabled={props.signalArray !== "" ? true : false} />
+                    <LabeledInput className="labels" type='number' label='Period(ms)' name='wave_period' value={deviceWavePeriod} onChange={changeWave_period} disabled={props.signalArray !== "" ? true : false} />
+                </div >
+            
+                <ChartComponent labelsArray={arr} dataArray={ar} chartName="Component Signal" />
```
Comment (YashodipD) [inline IoTDeviceSimulator/web-client/src/BehaviourComponent.js:54]: take this out save in JS variable and inside return that variable. Do similar things for other.

File: IoTDeviceSimulator/web-client/src/BehaviourComponent.js (hunk 1 above)
Comment (ashish-srivastava-dev) [reply IoTDeviceSimulator/web-client/src/BehaviourComponent.js:46]: This should be investigated in a subsequent PR.

File: IoTDeviceSimulator/web-client/src/BehaviourComponent.js (hunk 2 above)
Comment (ashish-srivastava-dev) [reply IoTDeviceSimulator/web-client/src/BehaviourComponent.js:54]: Address this comment in a subsequent PR.

**Review by ashish-srivastava-dev — APPROVED** [review]

**Pull request merged**